### DISCUSSION TITLE
Starting the bibliography counter from a given number

### DISCUSSION THREAD

**Discussion by arnaudgallou** [quarto-dev/quarto-cli#1476] [Q&A] [answered]
I have citations in a document (say doc2) that must follow citations from another document (doc1) sequentially, without displaying citations from doc1 in the bibliography. I then would like to set the counter of the bibliography in doc2 manually to make it starts at e.g. 46.

I've tried using `\setcounter{enumiv}{46}` in the `include-in-header` parameter based on a few posts (e.g. [here](https://tex.stackexchange.com/questions/408033/multibib-how-to-start-a-reference-counter-from-a-specific-number) or [there](https://tex.stackexchange.com/questions/289457/bibliography-starting-from-any-number)) but I can't make it work. The counter still starts at 1.

How can I set the counter in the bibliography?

**Comment by cderv** [comment, marked as answer]
Are you targetting PDF output here ? Did you customize [`cite-method`](https://quarto.org/docs/authoring/footnotes-and-citations.html#using-biblatex-or-natbib) configuration ?

If not it will use Pandoc citeproc (https://pandoc.org/MANUAL.html#citations) and your preamble won't be taken into account. 

You need to switch to natbib  probably to make it work.

**Reply by arnaudgallou** [reply]
Sorry, forgot to say it's for PDF. I didn't use the `cite-method` parameter so I initially tried it with the default value. I've just tried with `natbib` instead but it still doesn't work, whether I copy/paste codes I find online or just use `\setcounter{enumiv}{46}`.

**Reply by cderv** [reply]
I don't really know how to do that in LaTeX. Can you share a reproducible example so that I can have a look? What did you try exactly ?

**Reply by arnaudgallou** [reply]
I made a test repo [here](https://github.com/arnaudgallou/tmp). I didn't notice but I'm getting a `Misplaced alignment tab character &.` error when using `natbib` in the original document. The example I made runs properly with `natbib` which seems to ignore the `.csl`. I'm clueless about LaTeX, I tried the following:
```
include-in-header:
  text: |
    \setcounter{enumiv}{46}
```

```
include-in-header:
  text: |
    \usepackage{xpatch}
    \newcounter{mybibstartvalue}
    \setcounter{mybibstartvalue}{46}
    \xpatchcmd{\thebibliography}{%
      \usecounter{enumiv}%
    }{%
      \usecounter{enumiv}%
      \setcounter{enumiv}{\value{mybibstartvalue}}%
    }{}{}
```

```
include-in-header:
  text: |
    \usepackage{multibib}
    \let\oldthebibliography=\thebibliography
    \renewenvironment{thebibliography}[1]{%
       \oldthebibliography{#1}%
       \setcounter{enumiv}{46}%
    }
```
Tried with the default `cite-method` value, `biblatex` and `natbib`.

**Reply by cderv** [reply]
This is really a LaTeX issue. As you are targetting PDF and there is no supported config for that (as I think it is not that command to change it), you need to tweak the LaTeX directly. Probably the code you found is not working properly. Let me give a few details though: 

>  The example I made runs properly with natbib which seems to ignore the .csl

CSL file is only for Pandoc citeproc. So yes with `natbib` you'll need to use a `biblio-style` instead

> I'm clueless about LaTeX,

You'll need to search around and try - I found this: https://tex.stackexchange.com/questions/69675/how-to-set-the-number-of-the-first-citation

Try this 
````
---
format:
  pdf:
    cite-method: natbib
    keep-tex: true
    natbiboptions: numbers
    include-before-body:
      text: |
        \makeatletter
        \apptocmd{\thebibliography}{\global\c@NAT@ctr 15\relax}{}{}
        \makeatother
bibliography: references.bib
---

@brown1996, @vonhumboldt1805

```` 

It should work. 
* I use ` include-before-body` because `include-in-header` will insert before natbib is loaded, so I think that prevents it for working. 
* It using `etoolsbox` package but it is already included in the template. 

> I have citations in a document (say doc2) that must follow citations from another document (doc1) sequentially, without displaying citations from doc1 in the bibliography.

Maybe there is specific tool in LaTeX ecosystem that can help you with that, but I don't know it. 

Anwyay, I don't think Quarto can help here. You can ask your question on specific TeX Q&A site too.

**Reply by arnaudgallou** [reply]
Thank you so much for taking some time to look into it. Your code worked. The only issue I have now is to make it work with a specific citation style for the journal I'm submitting to but I should be able to get by from there.
